doc/sphinx/source/tutorial.rst

```diff
@@ -195,21 +163,21 @@ again. Let's dive into an example right away:
 
 .. code-block:: lua
 
-   use("aux", "coroutine")
+   include "scripting/coroutine.lua"
 
    function print_a_word(word)
       while 1 do
          print(word)
-         sleep(1000)
+         sleep(1000)         -- always sleep a bit, otherwise the system got stuck
```

Comment:
```suggestion
         sleep(1000)         -- always sleep a bit in loops, otherwise Widelands will hang
```

Reply:
It's not just widelands... i got sometimes there loosing control over the whole system. No mouse, no taskmanager, not able to switch windows... the only thing i could do was to press the main power button...

Reply:
I find this hard to believe since operating systems should surely never grant a single process full access to all the CPU power for more than a few milliseconds at a time? #4334 is another such weird thing. I'm not convinced that it is Widelands that effects such a total system freeze; I've seen a fair share of such freezes that seem to coincide suspiciously often with a particular program and yet cannot possibly be related.

I've tried out entering an endless loop in the debug console now, and while Widelands got hung up as expected, the rest of the system kept behaving just fine.

Reply:
Can't reproduce this in the debug console also. But surely i got there in the past while testing scenarios …

Anyway: The warning should be as hard as it can be, imho. It's both a situation which is bad, not all users know how to kill a process.

Reply:
Okay. But then how about putting this info prominently in a `.. note::` section instead of mentioning it on the side in a code example? E.g.
```
.. note:: 
   Always add :meth:`sleep` statements to lengthy loops.
   Widelands will hang while the code between two :meth:`sleep`
   calls is executed, resulting in a risk that the program – or
   even the whole operating system – may stall for an extended
   period of time, or even until Widelands is force-closed.
```

Reply:
Since there was already a not, i wrote a mix of my own words, the already written note and your suggestion.